src/sensor_ads1220.c

```diff
@@ -111,6 +119,15 @@ command_config_ads1220(uint32_t *args)
 DECL_COMMAND(command_config_ads1220, "config_ads1220 oid=%c"
     " spi_oid=%c data_ready_pin=%u");
 
+void
+command_attach_endstop_ads1220(uint32_t *args) {
+    uint8_t oid = args[0];
+    struct ads1220_adc *ads1220 = oid_lookup(oid, command_config_ads1220);
+    ads1220->lce = load_cell_endstop_oid_lookup(args[1]);
+}
+DECL_COMMAND(command_attach_endstop_ads1220, "attach_endstop_ads1220 oid=%c"
+    " load_cell_endstop_oid=%c");
```

Comment:
As before, I'd avoid using "endstop" in this command name - as it is not related to src/endstop.c .

In general, I think it would be preferable to prefix the commands with "ads1220" instead of as a suffix. I know we haven't been following that with a lot of the older commands, but going forward I think we should try.

Same comment for the hx71x.

Reply:
I think all of this is resolved now